docs/fundamentals-and-concepts/stellar-data-structures/assets.mdx

```diff
@@ -9,7 +9,7 @@ Accounts on the Stellar network can be used to track, hold, and transfer any typ
 
 Assets on Stellar have two identifying characteristics: the asset code and the issuer. Since more than one organization can issue a credit representing the same asset, asset codes often overlap (for example, multiple companies offer a USD token on Stellar). Assets are uniquely identified by the combination of their asset code and issuer.
 
-You can also create smart contract tokens using the [Token Interface](https://soroban.stellar.org/docs/reference/interfaces/token-interface) but it is possible, and even recommended in most cases, to [wrap a Stellar asset](#wrapping-stellar-assets) using the Stellar Asset Contract for use in smart contracts.
+You can also create smart contract tokens using the [Token Interface](https://soroban.stellar.org/docs/reference/interfaces/token-interface) but it is possible, and even recommended in most cases, to implement the [Stellar Asset Contract](#using-stellar-assets-in-smart-contracts) to interact with Stellar assets in smart contracts.
```

Comment:
I think this paragraph belongs in its own section at the very bottom of this page, a section titled "Custom Token Contracts".

And saying something like:
> Custom token contracts can be deployed on Stellar by deploying a contract that implements the [Token Interface](https://soroban.stellar.org/docs/reference/interfaces/token-interface), which is the same interface implemented by the Stellar Asset Contract for Stellar Assets.